paymentsheet/src/test/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEventTest.kt

```diff
@@ -22,6 +23,7 @@ import kotlin.time.Duration
 import kotlin.time.Duration.Companion.milliseconds
 import kotlin.time.Duration.Companion.seconds
 
+@OptIn(ExperimentalCardBrandFilteringApi::class)
```

Comment:
@jaynewstrom-stripe I'm wondering if the experimental annotations are in the right spot, it's requiring me to opt-in for all copy usage. Is this expected?

Reply:
Chatted over zoom, and worked this out!